src/RGUnityBots/Assets/Scripts/ActionManagerTests/UIHandler.cs

```diff
@@ -0,0 +1,80 @@
+﻿using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace ActionManagerTests
```

Comment:
All the files in this folder are scripts/components for a "test game" that is part of the ActionManagerTestScene. There are two problems with moving this into the test folder: 1) The scene would be referencing MonoBehaviours from inside the test assembly, which is an odd setup. More importantly, 2) this would cause these files to get ignored by the action analysis, which only targets the Player assemblies.

Reply:
Ah I see, thanks for the clarification